src/main/kotlin/org/elm/openapiext/Utils.kt

```diff
@@ -44,7 +45,11 @@ fun <T> Project.runWriteCommandAction(command: () -> T): T {
 }
 
 val Project.modules: Collection<Module>
-    get() = ModuleManager.getInstance(this).modules.toList()
+    get() = try {
+        ModuleManager.getInstance(this).modules.toList()
+    } catch (e: AlreadyDisposedException) {
+        emptyList()
+    }
```

Comment:
Is this fixing the problem “the intended way”, or is it hiding a deeper problem? Sorry if this is a stupid question, I have zero experience with Kotlin, Java and JetBrains development.

Anyhow, very exciting to see the CI go through! 🥳

Reply:
Very good point, I was just focusing on the symptoms instead of the cause. There is a `ApplicationManager.getApplication().invokeLater` call in the ElmWorkspaceService which failed when invoked after the test.
I changed the fix to ensure the async call is executed during the test instead of afterwards.